ui/src/features/stage/promotion-details-modal.tsx

```diff
@@ -188,6 +234,22 @@ export const PromotionDetailsModal = ({
       onOk={hide}
       onCancel={hide}
       cancelButtonProps={{ hidden: true }}
+      footer={
+        canAbortPromotion(promotion) && (
+          <>
+            <Button onClick={hide} type='primary'>
```

Comment:
Can we change the type to the default as everywhere for the cancel button?

Reply:
Its default everywhere else so I changed to default here